documentation/jira-guidelines.md

```diff
@@ -0,0 +1,132 @@
+- Acceptatiecriteria: het lijstje dat afgevinkt moet zijn voordat deze PBI klaar is.
+- Oplossingsrichting: als we al bedacht hebben welke stappen we willen doorlopen kunnen we die hier opvoeren. Ook zaken waar je aan moet denken, die in de refinement naar voren kwamen maar niet vergeten moeten zijn tegen de tijd dat je eraan begint.
+- Buiten scope: benoem aspecten die verwacht zouden kunnen worden maar die we later willen oppakken. Niet te zuinig, we willen PBI’s klein houden want groter worden ze vanzelf wel. Voorkomt premature discussies.
+- Actiepunten: als er nog dingen geregeld moeten worden, voorbereid, meetings ingepland. Mention degene die de actie gaat uitvoeren zodat dat duidelijk is die een notificatie krijgt.
+
+### Workflow
+
+Bij het aanmaken van een PBI is de state automatisch ‘New’. Alleen een titel en korte beschrijving zijn hier nodig, net voldoende om in de volgende state nog te begrijpen wat er moet gebeuren. Indien mogelijk ook al de juiste Epic en Labels koppelen.
+
+Wat er moet gebeuren om een PBI naar de volgende state te krijgen:
+
+- New
+  - Title en korte Description zijn er dus al.
+  - Leads voegen Rationale en Acceptatiecriteria toe.
+  - Waar nodig eventueel Epic en Labels aanvullen.
+  - Nog geen Assignee selecteren.
+- Ready to refine
+  - Team schrijft Oplossingsrichting, Buiten Scope, Actiepunten.
+  - Verfijnt wat er eventueel al stond.
+  - Leads bepalen prio op de backlog.
+- Ready for sprint
+  - De PBI is nu af en in principe moet de tekst niet meer bewerkt worden.
+  - Wel kunnen er Comments geschreven worden met relevante info die later ontstond.
+- Committed
+  - Zodra de PBI op een sprint komt wordt de state ‘Committed’.
+  - Dat verandert niet totdat alle Subtasks ‘Done’ zijn.
+  - Degene die het werk aan de PBI begint wordt de owner (assignee) van de PBI. Die neemt verantwoordelijkheid voor voortgang en afronding.
+- Completed
+
+### Subtasks
+
+Een Figma component story heeft in principe de volgende Subtasks:
+
+- Research
+  - Check status huidige component: Fabrique / onze Figma, onze Design System website.
+  - Kijk wat er beschikbaar is vanuit NL Design System.
+  - Zoek vergelijkbare ontwerpen in bijv. The Component Gallery
+  - Zet op een rij wat het component allemaal kan en hoe het eruit ziet.
+  - Ga na wat er aan a11y nodig is, bijv. Amsterdam TamTam, Fronteers Slack
+  - Betrek het perspectief van een front-ender en wellicht andere rollen (huisstijl, redactie?).
+  - Bepaal de naamgeving.
+- Design (eventueel meerdere Subtaken)
+  - Werk alle varianten van het component uit in Figma en zet ze overzichtelijk op een rij.
+  - Let op hover, focus states.
```

Comment:
Focus states aren't currently part of the design, right?

Reply:
Hmm… alright.